src/app/editor.js

```diff
@@ -147,16 +151,15 @@ function Editor (loadingFromGist) {
 
   function setupStuff (files) {
     var untitledCount = '';
-    if (!files.length || window.localStorage['sol-cache']) {
+    if (files.length === 0) {
       if (loadingFromGist) return;
       // Backwards-compatibility
```

Comment:
Actually we might not need that anymore - nobody used the old version at the new location.